### ISSUE TITLE
Support terrain packets

### ISSUE THREAD

**Issue by Gskartwii** [Arisstath/raknet-dissector#26]

**Comment by Gskartwii** [comment]
Blocked by #31 performance issues, must reconsider whether the GUI viewer should be included in beta-v0.8.

**Comment by Gskartwii** [comment]
Actually, I'm going to consider this "good enough" for now. A 3D viewer for terrain might be a bit out of scope. After all, a DataModel basepart viewer wouldn't make sense either. In the future, it might be good to have a way to serialize the terrain into dumped DataModels though.
